config/routes.rb

```diff
@@ -29,6 +29,7 @@
     end
   end
 
+  post '/index_urls' => 'index_urls#start'
```

Comment:
Let's follow the REST conventions. This should be using the `create` method. Also instead of `index_urls` we can use `urls`. So it's clear that this endpoint creates urls.